packages/react-router-config/README.md

```diff
@@ -78,6 +78,10 @@ const routes = [
 
 Returns an array of matched routes.
 
+#### Parameters
+- routes - the route configuration
+- pathname - the [pathname](https://nodejs.org/api/url.html#url_urlobject_pathname) component of the url
```

Comment:
Strictly speaking, this isn't a Node `urlObject.pathname`. The linked document specifies that a `urlObject.pathname` is not decoded, but the `pathname` provided to `matchRoutes` should be decoded. If it was not, then routes would not work with unicode characters. `history` automatically decodes the `pathname` when creating location objects.

I'm not sure if the best fix would be to change the link or just specify that the `pathname` should be decoded, but I think that it is a point worth clarifying.